contrib/python/pypodman/pypodman/lib/actions/_create_args.py

```diff
@@ -0,0 +1,394 @@
+        parser.add_argument(
+            '--expose',
+            metavar='RANGE',
+            help=('Expose a port, or a range of ports'
+                  ' (e.g. --expose=3300-3310) to set up port redirection.'),
+        )
+        parser.add_argument(
+            '--gidmap',
+            metavar='MAP',
+            help=('GID map for the user namespace'),
+        )
+        parser.add_argument(
+            '--group-add',
+            action='append',
+            metavar='GROUP',
+            help=('Add additional groups to run as'))
+        parser.add_argument('--hostname', help='Container host name')
+
+        volume_group = parser.add_mutually_exclusive_group()
+        volume_group.add_argument(
+            '--image-volume',
+            choices=['bind', 'tmpfs', 'ignore'],
+            metavar='MODE',
+            help='Tells podman how to handle the builtin image volumes')
+        volume_group.add_argument(
+            '--builtin-volume',
+            choices=['bind', 'tmpfs', 'ignore'],
+            metavar='MODE',
+            help='Tells podman how to handle the builtin image volumes')
+
+        parser.add_argument(
+            '--interactive',
+            '-i',
+            action=BooleanAction,
+            default=False,
+            help='Keep STDIN open even if not attached. (Default: False)')
+        parser.add_argument('--ipc', help='Create namespace')
+        parser.add_argument(
+            '--kernel-memory', action=UnitAction, help='Kernel memory limit')
+        parser.add_argument(
+            '--label',
+            '-l',
+            help=('Add metadata to a container'
+                  ' (e.g., --label com.example.key=value)'))
+        parser.add_argument(
+            '--label-file', help='Read in a line delimited file of labels')
+        parser.add_argument(
+            '--log-driver',
+            choices=['json-file', 'journald'],
+            help='Logging driver for the container.')
+        parser.add_argument(
+            '--log-opt',
+            action='append',
+            help='Logging driver specific options')
+        parser.add_argument(
+            '--memory', '-m', action=UnitAction, help='Memory limit')
+        parser.add_argument(
+            '--memory-reservation',
+            action=UnitAction,
+            help='Memory soft limit')
+        parser.add_argument(
+            '--memory-swap',
+            action=UnitAction,
+            help=('A limit value equal to memory plus swap.'
+                  'Must be used with the --memory flag'))
+        parser.add_argument(
+            '--memory-swappiness',
+            choices=range(0, 100),
+            metavar='[0-100]',
+            help="Tune a container's memory swappiness behavior")
+        parser.add_argument('--name', help='Assign a name to the container')
+        parser.add_argument(
+            '--network',
+            metavar='BRIDGE',
+            help=('Set the Network mode for the container.'))
```

Comment:
Set the Network mode for the container

"bridge", host, container:UUID, ns:PATH, none

Reply:
Added